packages/utils/src/PDFAbstractReference.js

```diff
@@ -0,0 +1,22 @@
+/*
+PDFAbstractReference by Devon Govett used below.
+The class is part of pdfkit. See https://github.com/foliojs/pdfkit
+LICENSE: MIT. Included in this folder.
+Modifications may have been applied for the purposes of node-signpdf.
+*/
+
+/*
+PDFAbstractReference - abstract class for PDF reference
+*/
+
+class PDFAbstractReference {
```

Comment:
Do we actually need this abstraction? Are there places where it is in use? Is there anything to abstract here?